stories/Utilities/FrostedImage/frosted-background.scss

```diff
@@ -20,7 +20,19 @@
   img {
     display: block;
     object-fit: cover;
+    opacity: 0.5;
   }
 }
 
+// If backdrop filter (blur) is supported, than apply the blurring effects.
+@supports (-webkit-backdrop-filter: none) or (backdrop-filter: none) {
+  .frosted-background {  
+    &::before {
+      @extend %frosted-bg;
+    }
+    img {
+      opacity: 1 !important;
```

Comment:
@paboden , can you please avoid `!important` rules here?

Reply:
fixed and pushed